libbeat/docs/loggingconfig.asciidoc

```diff
@@ -43,12 +43,12 @@ You can specify the following options in the `logging` section of the +{beatname
 
 ===== to_syslog
 
-Sends all logging output to syslog if enabled. For non-Windows systems, the default
-value is true.
+If enabled, sends all logging output to syslog (on non-Windows systems only). The default
```

Comment:
Ah, OK. I included it because I didn't think we would be able to write to a Windows syslog server, but it sounds like we would, so I will remove it.